roles/contiv_network/tasks/main.yml

```diff
@@ -35,6 +35,9 @@
 - name: copy systemd units for netplugin
   copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
 
+- name: copy bash auto complete file for netctl
+  shell: sudo cp /usr/bin/contrib/completion/bash/netctl /etc/bash_completion.d/ && sudo rm -rf /usr/bin/contrib
```

Comment:
can we have netplugin related files in the tar ball extracted as `netplugin/contrib/...`? I think we should namespace files per project as other project might also have similar directory names and it might not always be ok to delete those.

> Was just being skeptic about getting into a scenario where we have another netctl (auto completion file needs to have the same name as the cli binary) lying in inside /usr/bin, even if in separate folder inside.

I see (but how can there be two netctl binaries in /usr/bin?). 

The scenario I was worried about (and it might similar to yours), say user has netctlv2 but bash-completion is v1 (which can happen if user forgot to copy the file to /etc/bash_completion dir, after extracting the netplugin tar file). In that case it will be hard to debug a bash-completion anomaly.

On the other hand, say if we extract the netplugin related files to `usr/bin/netplugin/` dir and just create symlinks to these files instead. Now all the files are coming from same location, so as long as we extracted the netplugin tar correctly we don't need to worry about additional steps.

But I think both of our scenarios are pretty cooked up (and shall not happen as long as we assume ansible works correclty), so totally up to you if you want to make this change :)